plover/assets/main.json

```diff
@@ -49954,7 +49958,8 @@
 "PA*BG/STAPB": "pakistan",
 "PA*BGS": "PACS",
 "PA*BGT": "pact",
-"PA*D": "peripheral arterial disease",
+"PA*D": "{^pad}",
+"PA*D/WA": "Padua",
```

Comment:
How about `PAD/WA*` instead? Otherwise you might get:

`-T/TOUFP/PA*D/WA` for "the touch Padua" instead of "the touchpad was"